metrics/metrics.go

```diff
@@ -58,3 +65,86 @@ func (m *LabelMap) GetFloat(key string) *expvar.Float {
 func CurrentFDs() int {
 	return currentFDs()
 }
+
+type Histogram struct {
+	name string
```

Comment:
Seems like `name` is no longer used, and `Histogram` needs to be explicitly published?